roku_dev_cli/roku_dev_cli.py

```diff
@@ -119,7 +119,8 @@ def run(self):
                         if self.timestamp:
                             timestamp = datetime.datetime.now().strftime('%h %d, %Y %I:%M:%S%p')
                             line = bcolors.OKBLUE + "[" + timestamp + "] " + bcolors.ENDC + line
-                        print(line)
+                        if line:
```

Comment:
Is this checking against empty strings?

Reply:
Yep!  Python's empty string is falsy:  https://docs.python.org/3/library/stdtypes.html#truth-value-testing

Reply:
I understand that.  I'm just curious why this is needed now.  Is this related to the change, or unrelated just to catch additional empty lines from being printed?

Reply:
Ah – unrelated to the CPU change (hence the second commit).  I just wanted to get rid of the blank space while I was in the area.